index.js

```diff
@@ -0,0 +1,28 @@
+
+// Get the modal
+let modal = document.getElementById("modal");
```

Comment:
rename modal to something more semantic ie InstructionModal.
Use querySelector instead of .getElementByID